course_info/views.py

```diff
@@ -170,6 +187,33 @@ def editor(request):
     course_context = _course_context(request, _ORDERED_FIELD_NAMES, True,
                                      course_instance_id=course_instance_id)
     course_context['launch_presentation_return_url'] = \
-            request.POST.get('launch_presentation_return_url')
+        request.POST.get('launch_presentation_return_url')
     course_context['textarea_fields'] = ['description', 'notes']
     return render(request, 'course_info/editor.html', course_context)
+
+
+def sort_and_format_instructor_display(course_instructor_list):
+    """
+    Returns a sorted  string  sorted by  the instructors by role_id, seniority sort, last name
+    and then formatted such that it appears as a comma delimited String with an 'and' before that last user.
+    # Eg: User1, User2 and User3.
+    """
+    instructor_display = ''
+    # Note:  when seniority_sort is null, it was getting precedence over 1(eg: null, 1, 2).  So in such cases,
+    # it is being set to a large number (picked 100) so it is lower in the sorting order. [1, 2, ...null(set to 100)]
+    course_instructor_list.sort(key=lambda x: (x.get('role')['role_id'],
+                                               100 if x.get('seniority_sort') is None else x.get('seniority_sort'),
+                                               x.get('profile')['name_last']))
+
+    for person in course_instructor_list:
```

Comment:
Nope, it will disaply "Elliott Yates and Sapna Mysore" , as the 'and' replaces the right most ','. Quickly verified  this in unit tests and also via my local instance of the tool